views/event-detail.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+
+{% block mainContent %}
+  <div>
+    <p>What: {{event.title}}</p>
+    <p>When: {{event.date}}</p>
+    <p>Where: {{event.location}}</p>
+    <p>With whom: {{event.attending}}</p>
```

Comment:
Really close!  The outer tag should be `ul` and the inner within the for `li`.  Also, class is specified as regular attributes, not within it's own tag (e.g `<TAG class="event" id="eventDetail">`).

As for the line break, I'd guess it's related to the invalid markup, not certain.